activerecord/test/cases/fixtures_test.rb

```diff
@@ -64,7 +64,9 @@ def initialize
     end
 
     def call(_, _, _, _, values)
-      @events << values[:sql] if values[:sql] =~ /INSERT/
+      return unless values[:name] == "Fixtures Insert"
```

Comment:
I think the conditional insertion is a bit easier to read than the double-negative of an early exit on such a simple method.

(Why the change of condition?)